src/Smart.CFA.Pages/Pages/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,51 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>@ViewData["Title"] - Smart.CFA.Pages</title>
+    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
```

Comment:
Since we know we have a nice design system, it seems a bit sad to include almost 100 bootstrap-related files in this PR.

Reply:
Agreed, auto-generated in the templating of the app, but I see the point of not including them in the PR.